README.md

```diff
@@ -11,11 +11,11 @@ Package managers such as pip or apt usually inform the user the total disk space
 ### Specification
 
 - Input: dependency graph, existing packages, package to be installed
-- Output: the extra disk space needed 
+- Output: the extra disk space needed
 
 ### Assumptions
 
-This project assumes the user is using a GNU/Linux distribution on a x86_64 machine, and have all Python version needed on their machine. This would specify which package release the user should install, that is, `manylinux1_x86_64` if it is released, else `sdist`.
+This project assumes the user have all Python version needed on their machine. Moreover, it assumes all packages are distributed to any platforms and architecture. This would specify which package release the user should install, that is, `none-any`.
```

Comment:
This seems overly unrealistic. If we only do calculation, it's fine, but if we needs testing for it correctness then we have problem.

Reply:
Can rewrite it for me?

Reply:
You were the one rewriting it. I already wrote it to match my and your machines, as well and Long's and Huong's (their processors are AMD iirc which are also x86_64). You also have ubuntu installed which satisfy the Linux one.

One thing I did not consider, though, is the Python version. Afaik none of us have the latest version let alone all versions. Mine is 3.8, while the latest is 3.9?

Reply:
Okay. I guess I misunderstood back then.